TooltipDecorator/Tooltip.module.less

```diff
@@ -6,12 +6,12 @@
 
 .tail (@edge, @point) {
 	background-image:
-		linear-gradient(to @edge, @moon-tooltip-border-color @moon-tooltip-border-width, transparent @moon-tooltip-border-width), // Edge line
-		linear-gradient(to @point @edge, @moon-tooltip-bg-color 50%, @moon-tooltip-border-color 50%, @moon-tooltip-border-color ~"calc(50% + 1px)", transparent ~"calc(50% + 1px)"); // Diagonal line + fill
+		linear-gradient(to @edge, @moon-tooltip-border-color @moon-tooltip-border-width/2, transparent @moon-tooltip-border-width), // Edge line
+		linear-gradient(to @point @edge, @moon-tooltip-bg-color 50%, @moon-tooltip-border-color ~"calc(50% + 0.5px)", @moon-tooltip-border-color ~"calc(50% + 1px)", transparent ~"calc(50% + 1px)"); // Diagonal line + fill
 }
 .centerTail (@edge, @point) {
 	background-image:
-		linear-gradient(to @point @edge, @moon-tooltip-bg-color 50%, @moon-tooltip-border-color 50%, @moon-tooltip-border-color ~"calc(50% + 1px)", transparent ~"calc(50% + 1px)"); // Diagonal line + fill
+		linear-gradient(to @point @edge, @moon-tooltip-bg-color 50%, @moon-tooltip-border-color ~"calc(50% + 0.5px)", @moon-tooltip-border-color ~"calc(50% + 1px)", transparent ~"calc(50% + 1px)"); // Diagonal line + fill
```

Comment:
Do we have to change this part or what if we don't?

Reply:
Hello Jeonghee, here is what it looked like before. The top corners of the arrow were a liitle bit too deep in the tooltip body
![Without modification](https://user-images.githubusercontent.com/63335068/142383517-f30c1ef5-21a8-4096-b873-661782a78539.jpg)
And now:
![With modification](https://user-images.githubusercontent.com/63335068/142383531-3941beb0-44cc-429c-a8b2-63a3fa4a66a9.jpg)
If you think this change is not necessary, let us know

Reply:
I understand. Thanks for the kind explanation.